net/speedtest/speedtest.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package speedtest
+
+import (
+	"fmt"
+)
+
+const (
+	Start = "start" // Start the test.
+	End   = "end"   // End the test.
+	Data  = "data"  // Message contains data.
+
+	LenBufJSON    int = 100   // agreed upon before hand. Buffer size for json messages.
```

Comment:
don't need types

see https://blog.golang.org/constants

but does this need to be exported?